source/jormungandr/tests/bragi_autocomplete_tests.py

```diff
@@ -1667,6 +1667,25 @@ def test_global_coords_uri_new_bragi_coord_id_fmt(self):
             assert address['id'] == '3.282103;49.847586'
             assert len(address['administrative_regions']) == 1
 
+    def test_geo_status(self):
+        """
+        geo_status is still not implemented for bragi
+        we test the return code and error message
+        """
+        url = 'https://host_of_bragi'
+        params = {'pt_dataset[]': 'main_routing_test', 'lon': 3.282103, 'lat': 49.84758, 'timeout': 200}
+
+        url += "/reverse?{}".format(urlencode(params, doseq=True))
```

Comment:
I don't see any use of this line same as above 'url = 'https://host_of_bragi' '

Reply:
fixed, thanks @kadhikari